testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -432,7 +447,7 @@ public void createCatalog(String catalogName, String connectorName, Map<String,
             catalogNames.add(server.createCatalog(catalogName, connectorName, properties));
         }
         CatalogName catalog = getOnlyElement(catalogNames);
-        log.info("Created catalog %s (%s) in %s", catalogName, catalog, nanosSince(start));
+        log.info("Created catalog %s (%s) in %s, with properties: %s", catalogName, catalog, nanosSince(start), properties);
```

Comment:
Logging properties in `DistributedQueryRunner` is dangerous for SaaS connector. We should consider other way. e.g. Move logging place to other places, prepare toggle and disable by default and so on.

Reply:
Thanks, that would be a serious issue. I moved this to the SqlServerQueryRunner.

Reply:
We should log properties via Bootstrap since it's aware of ConfigSecuritySensitive and can mask logs properly. Anything else seems error prone.

Reply:
We're talking about tests here only. I reverted this change in the `DistributedQueryRunner`.